dialog/file.go

```diff
@@ -17,11 +17,12 @@ import (
 	"fyne.io/fyne/v2/widget"
 )
 
-type viewLayout int
```

Comment:
Public comments are needed for all the new exported types and constants, including the "Since: 2.5" line.

Reply:
thanks! Done

Reply:
FYI: It is common practice for us to let the one opening the review comment mark them as done and not the one fixing them. It is fine for now but it is worth noting.

Reply:
> FYI: It is common practice for us to let the one opening the review comment mark them as done and not the one fixing them. It is fine for now but it is worth noting.

Sure, thanks for the feedback. I am still learning the open-source contribution etiquette and feedback like this helps me get better

Reply:
No problem. Happy to guide you :)